save-orchestrator/build.gradle.kts

```diff
@@ -20,7 +22,7 @@ tasks.withType<KotlinCompile> {
 }
 
 tasks.getByName("processResources").finalizedBy("downloadSaveCli")
-val saveCliVersion: String = if (Versions.saveCore.endsWith("SNAPSHOT")) {
+val saveCliVersion: String = if (Versions.saveCore.endsWith("SNAPSHOT") || Versions.saveCore.endsWith("]")) {
```

Comment:
Why ?
```kotlin
Versions.saveCore.endsWith("]"))
```

Reply:
I was experimenting with version ranges to use specific snapshot version and forgot to delete this, I'll delete it now